File: totosuki/ABC/B/061.py
N, M = map(int, input().split())
rslt = [[0]*N for _ in range(N)]
for _ in range(M):
  a, b = map(int, input().split())
  rslt[a-1][b-1] += 1
  rslt[b-1][a-1] += 1
for row in range(N):
  cnt = 0
  for col in range(N):
    cnt += rslt[row][col]
  print(cnt)

File: OCH_Tao/ABC/B/061.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const [N, M] = ((input.shift()).split(" ")).map(x => parseInt(x, 10));
let l = [];
for (let i = 0; i < M; i++) {
  let [a, b] = (input[i].split(" ")).map(x => parseInt(x, 10));
  l.push(a);
  l.push(b);
}
for (let i = 1; i < N + 1; i++) {
  let cnt = 0;
  for (let j = 0; j < l.length; j++) {
    if (l[j] === i) {
      cnt++;
    }
  }
  console.log(cnt);
}